app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -3,7 +3,7 @@
 
   .Discussion,
```

Comment:
Should I address this? And how? Moving the `margin` to `.Discussion` and `.CommentCard`, or using `.CardList:nth-child(n)`?